cli_reference/basic_cli_operations.adoc

```diff
@@ -467,6 +467,20 @@ $ oc run NAME --image=<image> \
     [options]
 ----
 
+=== patch
+Updates one or more fields of an object using strategic merge patch:
+----
```

Comment:
Hmm the rest of the file is like this, and this is the first i'm hearing of this being a thing. I'll leave for now, but maybe I'll do some comparisons one day...